frontend/src/components/TooltipIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Tooltip from '@material-ui/core/Tooltip';
+import classNames from 'classnames';
+import css from './TooltipIcon.module.scss';
+
+export const TooltipIcon: React.FC<{
+  title: string;
+  children?: any;
+}> = ({ title, children }) => {
+  return (
+    <Tooltip
```

Comment:
```suggestion
    <StylesProvider injectFirst>
      <Tooltip
```

The custom styles didn't seem to work, using styles provider should help (from @material-ui/core/styles)

Reply:
This is no longer necessary, as the styles provider is used on the whole application